<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="buscarbtn">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUbicacion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ubicacion</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>100</y>
        <width>271</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLeer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMostrar">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>380</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFuerza">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Fuerza bruta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOrdenar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordenar ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCoordenadas">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordenadas</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>467</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="3" colspan="2">
        <widget class="QLineEdit" name="txtPosY"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Posicion X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="txtVoltaje"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="txtId"/>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="btnGuardar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>GUARDAR AL INICIO</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLineEdit" name="txtBlue"/>
       </item>
       <item row="3" column="0" rowspan="2">
        <widget class="QLineEdit" name="txtPosX"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="txtGreen"/>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="btnGuardarFinal">
         <property name="text">
          <string>GUARDAR AL FINAL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Voltaje</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>RED</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>BLUE</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="txtRed"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>GREEN</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" rowspan="2" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Posicion Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnEscribir">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Escribir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimpiar">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>420</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOrdenarVoltaje">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>380</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordenar Voltaje</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>701</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans Ultra Bold Condensed</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(11, 19, 48);</string>
      </property>
      <property name="text">
       <string>NEURONAS</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>140</y>
        <width>441</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tap_Grafo">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGraphicsView" name="viewGrafo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>791</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GRAFO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VerLista">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver lista</string>
      </property>
     </widget>
     <widget class="QPushButton" name="verGrafo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Grafo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="limpiarGrafo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txtListaAdyacencia">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>511</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="generarGrafo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBusqueda">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Busqueda</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBusqueda">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionguardar"/>
   </widget>
   <widget class="QMenu" name="menuGrafo">
    <property name="title">
     <string>Grafo</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuGrafo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionguardar">
   <property name="text">
    <string>guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
